kubernetes-examples/pom.xml

```diff
@@ -39,7 +39,12 @@
       <groupId>io.fabric8</groupId>
       <artifactId>openshift-client</artifactId>
     </dependency>
-
+    
+    <dependency>
+      <groupId>com.squareup.okhttp3</groupId>
+      <artifactId>okhttp</artifactId>
+    </dependency>
+    
```

Comment:
Isn't this provided transitively by the kubernetes-client dependency (kubernets-client <- kubernetes-httpclient-okhttp <- okhttp3)?

Reply:
Only at a runtime scope.  There are compile references in the examples as well.

Reply:
OK, shall we mark this as `provided`?

Reply:
No, PortForwardExample is creating it's own client, so it's a hard dependency.

Reply:
OK, I see now.
I guess that to avoid confusion, we can refactor (in the future) the PodForwardExample to use something else (maybe just a plain URL connection.